frt-part-two.md

```diff
@@ -0,0 +1,40 @@
+| Author | TallAmericano |
+| :---- | :---- |
+| Title | The Face of Tomorrow: Exploring the Upsides and Downsides of Facial Recognition Technology (FRT) (2/2) |
```

Comment:
Anyone up for brainstorming a title for this piece?

My initial thought is to lose the "(FRT) (2/2)" and shorten it to "The Face of Tomorrow"

Reply:
Agreed I'll change this also. But maybe that means article 1 will need to change too to be consistent. Or Maybe 'The Face of Tomorrow Part One' and 'The Face of Tomorrow Part Two'